comm/src/main/scala/coop/rchain/catscontrib/ski/package.scala

```diff
@@ -0,0 +1,10 @@
+package coop.rchain.catscontrib
+
+package object ski {
+  def σ[A, B, C](x: A => B => C, y: A => B, z: A): C         = x(z)(y(z))
```

Comment:
It's even more fun if you write:
```scala
def σ[A, B, C](x: A => B => C, zz: A => B, y: A): C         = x(y)(zz(y))
```
And it also makes it clear that this is magic.

Reply:
Hahaha!